app/models/qernel/graph_api/energy.rb

```diff
@@ -101,10 +101,24 @@ def total_number_of_excess_events(excludes = [])
       # Public: Returns number of blackout hours
       #
       # Returns an Integer
-      def number_of_blackout_hours
+      def number_of_power_shortage_hours
         graph.plugin(:merit).order.blackout.number_of_hours
       end
 
+      # Public: Returns the peak of blackout hours, defined as the largest single hour deficit
```

Comment:
Following our discussion that documentation should be useful for both modellers and developers: it would be useful to add units to the comments of these functions: is this MJ, MW, MWh?